web-client/public/js/setuphandlers.js

```diff
@@ -0,0 +1,11 @@
+import grnState from "grnstate";
+import updateApp from "updateapp";
+export var setupHandlers = function (grnState) {
+    var updateSliderDisplayedValue = function (slider, element) {
+        var value = $("#" + $(element).attr("id")).val();
+        $(slider.valueId).html(value + ((slider.needsAppendedZeros &&
+          (value.length === grnState.GRAVITY_LENGTH_WITHOUT_ZERO)) ? "0" : ""));
+        slider.setCurrentVal(value);
+    };
```

Comment:
This is a helper function that looks more like it belongs to the updateApp function rather than setupHandlers. Notice what it does: it updates what an element on the page is displaying. That's what updateApp is for. setupHandlers is for the _opposite_  direction: it sets up what to do when the user interacts with the page. So setupHandlers should be for stuff like `click`, `change`, and `input`. Anything that goes _toward_ a visible UI element belongs in updateApp.